.storybook/main.js

```diff
@@ -39,6 +40,12 @@ module.exports = {
       ],
     });
 
+    // Update @storybook/addon-docs webpack rules to load all .mdx files in storybook
+    const mdxRule = config.module.rules.find(rule => rule.test.toString() === /\.mdx$/.toString());
+    mdxRule.use.find(loader => loader.loader.includes('@mdx-js')).options['compilers'] = [
+      createCompiler({}),
+    ];
```

Comment:
This change makes it so that every `.mdx` file found in the `stories` field above will be processed for Storybook and rendered within Storybook. This means that every `.mdx` file will require a `<Meta />` tag otherwise you will get an error in Storybook (something about a default export with no title). Previously, storybook would only render `.mdx` files with the `.story` or `.stories` prefix.

To get around this, anything you'd like visible in github, but not in Storybook should be `.md`:

- Github only: `.md`
- Storybook + Github: `.mdx` with `<Meta />`